_posts/2018-04-18-finance-manager.md

```diff
@@ -4,10 +4,13 @@ title: Finance Manager
 tags: draft
 ---
 
-Bridge Foundry is seeking a Finance Manager (part-time contract).
-This position is a great opportunity to help shape the success of our growing
-nonprofit organization, supporting the work of hundreds of volunteers who
-organize workshops, ...
+Know someone who wants to change the world and keep the books? Bridge Foundry is seeking a part-time Finance Manager.
+
+We are looking for someone who will help shape the success of our growing nonprofit organization, supporting the work of hundreds of volunteers who organize workshops around the world.
```

Comment:
who organize tech training workshops, raise funds to send folks to conferences, and other initiatives that create high impact change.